manuscript.md

```diff
@@ -36,6 +36,20 @@
 
 ## What is a species interaction?
 
+Network ecologists have primarily focused their research on food webs 
```

Comment:
This section might actually be more suited to what is limiting us in the meta...

Reply:
there are also possibly some sweeping statements contained in this paragraph...